testsuite/forge-cli/src/main.rs

```diff
@@ -259,11 +274,24 @@ fn main() -> Result<()> {
                     if let Some(move_modules_dir) = &k8s.move_modules_dir {
                         test_suite = test_suite.with_genesis_modules_path(move_modules_dir.clone());
                     }
+                    let namespace = if k8s.namespace.is_none() {
+                        let mut rng : ThreadRng= rand::thread_rng();
+                        let words = std::fs::File::open("/usr/share/dict/words")
```

Comment:
It probably makes sense to hard-code a sanitized list of words, and append a short (6-8 chars) random string. The `words` dict has plenty of NSFW words.

Reply:
I personally dont see a huge problem with that, however I can pick words of a certain length

Reply:
does this work across OS?